src/ros_can_nodes/include/CANBuffers.hpp

```diff
@@ -0,0 +1,47 @@
+/*
+ * Date Started: 20/10/2017
+ * Original Author: Simon Ireland
```

Comment:
is this right?

Reply:
technically yes. this was renamed from TopicBuffers, but i pretty much overwrote the whole thing with this refactor